django_mongodb_engine/compiler.py

```diff
@@ -129,7 +132,7 @@ def order_by(self, ordering):
     @safe_call
     def delete(self):
         options = self.connection.operation_flags.get('delete', {})
-        self.collection.remove(self.mongo_query, **options)
+        self.collection.delete_many(self.mongo_query, **options)
```

Comment:
@kondaiahv : how to take care of multi=True?

Reply:
delete_many is delete + multi=true

Reply:
collection.remove({'a': 1}, multi=False)
collection.remove({'b': 1})
Can be changed to this:

collection.delete_one({'a': 1})
collection.delete_many({'b': 1})

what I think is self.collection.remove(self.mongo_query, **options) --> this is handling both  single and multi, if thats the case then we need to support both delete_one and delete_many

Reply:
options value is empty. We need to check how we are using delete() operations in Django models.

Reply:
Looking at the api-server code, there is no remove() function call with multi=False.
So changing to delete_many() is fine.